.github/workflows/build-and-test-toolchain.yml

```diff
@@ -140,14 +137,12 @@ jobs:
 
       - name: Execute GCC tests
         run: |
-          cd ~/work
-          .github/scripts/toolchain/execute-gcc-tests.sh "${{ env.TAG }}" "${{ env.MODULE }}" "${{ env.FILTER }}"
+          ~/work/.github/scripts/toolchain/execute-gcc-tests.sh "${{ env.TAG }}" "${{ env.MODULE }}" "${{ env.FILTER }}"
 
       - name: Create summary
         run: |
-          cd ~/work
-          source .github/scripts/config.sh
-          .github/scripts/toolchain/create-gcc-summary.sh "${{ env.TAG }}" >> \
+          source ~/work/.github/scripts/config.sh
```

Comment:
Not sure config.sh is needed here

Reply:
It is needed, otherwise \`$ARTIFACT\_PATH\` would be empty.

Reply:
maybe it makes sense to move ARTIFACT_PATH to the script and make it as the root for summary. if the root should be different, than define ARTIFACT_PATH outside the script.

Reply:
That would be counter-intuitive. It's much common to specify output directory as a script argument if the script should be used sandalone rather that controll it with environment vairable. What would be the benefit outside of this specific WSL environment context?

Reply:
That it does not have to be defined and will be default in the most cases. It is a suggestion and both options are fine for me.

Reply:
OK, I'll do this particular change within https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/pull/186 as it belongs there.

Reply:
Hmm, that won't work either because bounding `create-gcc-summary.sh` to `$ARTIFACT_PATH` disallows using it on directly the GCC sources after the tests were executed.

Reply:
I personally prefer to have the path as argument rather than passed in env var even though we have to `source` because of it. So the fact that it has to stay is no bother for me